src/main/kotlin/dev/stalla/parser/namespace/PodcastindexParser.kt

```diff
@@ -87,7 +83,10 @@ internal object PodcastindexParser : NamespaceParser() {
             .type(type)
     }
 
-    private fun Node.toSoundbiteBuilder(soundbiteBuilder: EpisodePodcastindexSoundbiteBuilder): EpisodePodcastindexSoundbiteBuilder? {
+    @Suppress("ReturnCount")
```

Comment:
Again, not better idea for this method.

Reply:
It's ok, it was already the smallest/simplest implementation in my mind 👯